### PR TITLE
cluster-init: Add `provision ocp [create-install-config|create-manifests|create-cluster]` commands

### PR DESCRIPTION
**Pull request by danilo-gemoli**
Add three `provision` sub-commands:
1. `cluster-init provision ocp create-install-config`: execute `openshift-install create install-config`
2. `cluster-init provision ocp create-manifests`: execute `openshift-install create manifests`
3. `cluster-init provision ocp create-cluster`: execute `openshift-install create cluster`

Those commands are so similar in structure that I have decided to put them all in a single PR. Each of them performs what follows:
- read the `cluster-install.yaml`
- prepare a command: `exec.Cmd{}`
- run `openshift-install ...`

### CODE REVIEW

File: cmd/cluster-init/cmd/provision/ocp.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			}
+			return nil
+		},
+	}
+	return &cmd
+}
+
+func newOCPCreateManifests(ctx context.Context, log *logrus.Entry) *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "create-manifests",
```
Comment (deepsm007) [inline cmd/cluster-init/cmd/provision/ocp.go:61]: nit `"create manifests",`

File: cmd/cluster-init/cmd/provision/ocp.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+			}
+			return nil
+		},
+	}
+	return &cmd
+}
+
+func newOCPCreateCluster(ctx context.Context, log *logrus.Entry) *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "create-cluster",
```
Comment (deepsm007) [inline cmd/cluster-init/cmd/provision/ocp.go:77]: `create cluster`

File: cmd/cluster-init/cmd/provision/ocp.go (hunk 1 above)
Comment (danilo-gemoli) [reply cmd/cluster-init/cmd/provision/ocp.go]: Nice, will do.

**Pull request merged**